src/data/automation_i18n.ts

```diff
@@ -60,18 +60,32 @@ export const describeTrigger = (
 
     base += ` ${entity} is`;
 
-    if ("above" in trigger) {
+    if (trigger.above !== undefined) {
       base += ` above ${trigger.above}`;
     }
 
-    if ("below" in trigger && "above" in trigger) {
+    if (trigger.below !== undefined && trigger.above !== undefined) {
       base += " and";
     }
 
-    if ("below" in trigger) {
+    if (trigger.below !== undefined) {
```

Comment:
Hmm this might indicate a different bug,m (which this would be working around) why isn't is removed from the trigger?

Reply:
You're right, there is definitely an existing bug here. The object should be identical before and after entering and removing values from the "before" and "after" fields. But instead, this is what I observe:

Before:
![image](https://user-images.githubusercontent.com/2292715/194773782-74697ed5-42fe-4a6a-99de-9cfd1468d838.png)

After entering values and then removing them from the "before" and "after" fields:
![image](https://user-images.githubusercontent.com/2292715/194773812-94fd4c85-bb2f-4369-b472-f29ab4498b57.png)

IMHO, even if this bug is fixed, the change in this PR should still be made. The current logic is incomplete because it just checks for a missing value, not an undefined value.